nemo/collections/llm/gpt/model/ssm.py

```diff
@@ -705,7 +707,10 @@ def config(self):
             model_path = local_model_path if local_model_path else "nvidia/Nemotron-H-56B-Base-8K"
             hf_config = AutoConfig.from_pretrained(model_path, trust_remote_code=True)
         elif type(source) == NemotronNano9Bv2:
-            model_path = local_model_path if local_model_path else "nvidia/NVIDIA-Nemotron-Nano-9B-v2"
+            model_path = local_model_path if local_model_path else "nvidia/NVIDIA-Nemotron-Nano-9B-v2-Base"
+            hf_config = AutoConfig.from_pretrained(model_path, trust_remote_code=True)
+        elif type(source) == NemotronNano12Bv2:
```

Comment:
Wonder if we could just take `hf_config = AutoConfig.from_pretrained(model_path, trust_remote_code=True)` out from the if branch and put it at the end since its the same for all conditions